iOS/DuckDuckGo/NewTabPageShadowScrollView.swift

```diff
@@ -21,7 +21,7 @@ import SwiftUI
 import UIKit
 import DesignResourcesKit
 
-struct NewTabPageShadowScrollView<Content: View>: UIViewRepresentable {
+struct NewTabPageShadowScrollView<Content: View>: UIViewControllerRepresentable {
```

Comment:
Because it's required to host a SUI view inside the scroll view, the `UIHostingController` which does that needs to have a parent. There's no other way to reliably do this other than use the VCRepresentable.